mappings/net/minecraft/entity/Living.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/sd net/minecraft/entity/Living
```

Comment:
This class is the interface that *all* living entities implement. It doesn't really do anything so it's hard to give it an accurate name. Didn't know we were keeping the I prefix though, will change that.